rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -2247,6 +2247,22 @@ public byte[] getPendingFederatorPublicKeyOfType(int index, FederationMember.Key
         return federationSupport.getPendingFederatorPublicKeyOfType(index, keyType);
     }
 
+    /**
+     * Retrieves the Bitcoin address of the proposed federation, if it exists.
+     *
+     * <p>
+     * This method checks if there is a proposed federation available and 
+     * returns its associated Bitcoin address. The proposed federation is typically 
+     * a federation that is awaiting approval or activation.
+     * </p>
+     * 
+     * @return an {@link Optional} containing the Bitcoin {@link Address} of the proposed federation,
+     *         or an empty {@link Optional} if no proposed federation is available.
+     */
```

Comment:
I think this would add more value in FederationSupport interface rather than here. We actually aim to eventually remove this and call FederationSupport directly

Reply:
I see. I have moved the javadoc to the FederationSupport impl.